package.json

```diff
@@ -17,6 +17,9 @@
     "bem",
     "typography"
   ],
+  "engines": {
+    "node": ">=12.0.0"
```

Comment:
This was needed because eslint was complaining about `fs.promises` being used and it not being supported until node 11. By specifying the minimum node version here, the rule does not trigger